ajax/libs/pwnjs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pwnjs",
+  "filename": "dist/pwn.min.js",
```

Comment:
Just use `pwn.min.js` here. You can regard `basePath` in auto-update config as a root.